src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CommandResolverWithVectorizerTest.java

```diff
@@ -118,6 +138,39 @@ public void find() throws Exception {
               });
     }
 
+    @Test
+    public void findNonVectorize() throws Exception {
+      String json =
+          """
+          {
+            "find": {
+              "sort" : {"$vectorize" : "test data"}
+            }
+          }
+          """;
+
+      FindCommand findOneCommand = objectMapper.readValue(json, FindCommand.class);
+
+      Throwable throwable =
+          dataVectorizerService
+              .vectorize(VECTOR_COMMAND_CONTEXT, findOneCommand)
+              .subscribe()
+              .withSubscriber(UniAssertSubscriber.create())
+              .getFailure();
+
+      assertThat(throwable)
+          .isInstanceOf(JsonApiException.class)
+          .satisfies(
+              e -> {
+                JsonApiException exception = (JsonApiException) e;
+                assertThat(exception.getMessage())
+                    .isEqualTo(
+                        "Unable to vectorize data, embedding service not configured for the collection");
+                assertThat(exception.getErrorCode())
+                    .isEqualTo(ErrorCode.UNAVAILABLE_EMBEDDING_SERVICE);
```

Comment:
I guess names can be whatever, but "UNAVAILABLE" sounds like availability issue -- so should it rather be something like "EMBEDDING_SERVICE_NOT_CONFIGURED" or "NO_EMBEDDING_SERVICE_FOR_COLLECTION"?